src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1446,12 +1450,24 @@ because xbuild doesn't support framework reference assemblies.
     _PrepareAssemblies;
     _PrepareNativeAssemblySources;
     $(_AfterPrepareAssemblies);
+    _GetGenerateJavaStubsInputs;
   </_GenerateJavaStubsDependsOnTargets>
 </PropertyGroup>
 
+<Target Name="_GetGenerateJavaStubsInputs">
+  <ItemGroup>
+    <_GenerateJavaStubsInputs Include="@(_AndroidMSBuildAllProjects)" />
+    <_GenerateJavaStubsInputs Include="$(_ResolvedUserAssembliesHashFile)" />
+    <_GenerateJavaStubsInputs Include="@(_ResolvedUserMonoAndroidAssemblies)" />
+    <_GenerateJavaStubsInputs Include="$(_AndroidManifestAbs)" />
+    <_GenerateJavaStubsInputs Include="$(_AndroidBuildPropertiesCache)" />
+    <_GenerateJavaStubsInputs Include="@(AndroidEnvironment);@(LibraryEnvironments)" Condition=" '$(_AndroidFastDeployEnvironmentFiles)' == 'false' " />
```

Comment:
I believe the advice was that one shouldn't compare `== 'false'`, but instead should use `!= 'true'`.